sources/packages/web/src/components/institutions/modals/confirmationOfEnrollment/ApproveCOE.vue

```diff
@@ -54,7 +54,6 @@
           primaryLabel="Continue to confirmation"
           @primaryClick="submit"
           @secondaryClick="cancel"
-          :disablePrimaryButton="notAllowed"
```

Comment:
Can you please elaborate on why was this removed?

Reply:
I believe that assignment of this attribute is a dead code as notAllowed is not a property that is defined in the component.

Reply:
it was a spillover. this is an institution component